migrate2rocky/README.md

```diff
@@ -16,15 +16,23 @@ Running this script will convert an existing CentOS 8 system to Rocky Linux 8.
 
 ### Known Issues
 
-#### Katello clashes
-
-Katello installs its own repositories via subscription-manager that are meant to
-replace the ones from CentOS, but it does not remove, or even disable, the
-corresponding CentOS repositories.  This would cause issues to a normal running
-system, but it is especially problematic for migrate2rocky as it means that
-migrate2rocky cannot properly determine which repositories to remove and
-attempting to run migrate2rocky on a katello will likely result in a corrupted
-system.
+#### Custom replacements of default repositories
+
+This script expects the **original repository configuration being present, as well
+as enabled** (i.e. for CentOS the `baseos` repo configuration in the
+`/etc/yum.repos.d/CentOS-Linux-BaseOS.repo` file has to be present and enabled).
+Also make sure that there are **no other repositories** which could interfere with the
+original configuration.
+
+As long as the original repository configuration is disabled and a new alternative
+repository gets added in a different file, or not the only repo that provides the
+base packages, the migration result will end in a corrupted system.
+
+This especially happens on systems configured with a centralized package management
+like Katello (RedHat Satellite 6) or Uyuni (RedHat Satellite 5, SUSE Manager).
+If one of these systems is configured it is possible to remove all projects/channels
+so the designated /etc/yum.repos.d/redhat.repo or
+/etc/yum.repos.d/susemanager:channels.repo file is empty or missing.
```

Comment:
redhat.repo is in place and not empty for any distribution (at this stage only RHEL) or any software that makes use of subscription-manager.  Such installations are supported by migrate2rocky so long as they don't make alterations to, duplicate, or override the core system repositories that come with the original distribution.  Attempting to determine if a migration is supported, or will work, based on the presence of this file or the presence or lack of content in it is broken because it will lead to migrations from from RHEL 8 not working.

In addition to this, there should be language like the following:  "Any distribution that has had its core repositories altered, removed, duplicated or overridden may cause migrate2rocky to break or corrupt the system when run.  Any attempt to migrate such systems, even after reversing the changes made by such software, is not supported in any way.  In all cases you should backup your system before using migrate2rocky and USE AT YOUR OWN RISK."

I especially want that type of language in place if we're going to give any type of instructions on how to migrate katello or similar systems.